samples/sqlservices/package.json

```diff
@@ -62,7 +69,9 @@
         "postinstall": "node ./node_modules/vscode/bin/install && node ./node_modules/sqlops/bin/install && gulp copytypings"
     },
     "dependencies": {
-        "vscode-nls": "^3.2.2"
+        "vscode-nls": "^3.2.2",
+        "fs-extra": "^5.0.0",
+        "handlebars": "^4.0.11"
```

Comment:
What are the new dependencies for? I don't see them used elsewhere in this change

Reply:
there were missing and made the extension fail to run

---
In reply to: [188376151](https://github.com/Microsoft/sqlopsstudio/pull/1415#discussion_r188376151) [](ancestors = 188376151)